src/runtimeMain/kotlin/com/epam/drill/test/agent/instrumentation/http/selenium/Selenium.kt

```diff
@@ -138,28 +149,24 @@ class Selenium : TransformStrategy() {
         ).insertBefore(
             """
                 if($1.equals(org.openqa.selenium.remote.DriverCommand.SWITCH_TO_WINDOW)){
-                   ${connectToDevTools()}
+                    drillDevTools.${ChromeDevTool::switchSession.name}(getCurrentUrl());
```

Comment:
It confuses me that there is no check for null, of course, calling webdriver methods without starting a session is nonsense, but if this is possible, you will break everything, can you still play it safe and check?
In general, what was the point of move it into the fields?